docs/product/components/user-cards.html

```diff
@@ -10,8 +10,8 @@
 {% highlight html %}
 <div class="s-user-card">
     <time class="s-user-card--time">3 mins ago</time>
-    <a href="#" class="s-user-card--avatar s-avatar s-avatar__md">
-        <img class="bar-sm" src="https://placekitten.com/32/32" />
+    <a href="#" class="s-avatar s-avatar__md s-user-card--avatar">
+        <img class="s-avatar--image" src="https://placekitten.com/64/64" />
```

Comment:
Really appreciate this change especially, I think updating s-avatar will be a big help overall